.github/workflows/builds.yml

```diff
@@ -49,3 +49,23 @@ jobs:
           pip install -e .[torch] --upgrade
       - name: Import package
         run: python -c "import doctr; print(doctr.__version__)"
+
+  conda:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v3
+        with:
+          auto-update-conda: true
+          python-version: 3.9
+      - name: Install dependencies
+        shell: bash -el {0}
+        run: conda install -y conda-build conda-verify anaconda-client
+      - name: Build and publish
+        shell: bash -el {0}
+        run: |
+          echo "BUILD_VERSION=0.8.0a0" | cut -c 2- >> $GITHUB_ENV
+          python setup.py sdist
+          mkdir conda-dist
+          conda build .conda/ -c conda-forge -c pypdfium2-team -c bblanchon --output-folder conda-dist
```

Comment:
... these channels will also have to be added on the end users' install command, which is tedious - sadly custom channels are second class with conda ...

Reply:
an entry for the installation docs section please then it should be fine :sweat_smile:

Reply:
still it's a bit sad end users can't just do `conda install doctr`
but in any case, they'll have to specify the channel you'll be publishing to, so hopefully two more custom channels shouldn't be that much of a problem